packages/rainbow-button/src/components/RainbowButton.tsx

```diff
@@ -10,8 +11,8 @@ export function RainbowButtonProvider({
   'chains' | 'avatar' | 'initialChain' | 'modalSize' | 'showRecentTransactions'
 >) {
   return (
-    // chains array is intentionally left empty
-    <RainbowKitProvider chains={[]} {...options}>
+    // Wagmi requires 'chains' as a tuple type with at least one chain
+    <RainbowKitProvider chains={[mainnet]} {...options}>
```

Comment:
This should only be consumed within RainbowKit, and not passed to Wagmi, right? Do we need to add a chain here?

Reply:
Yeah good point. Wagmi v2 uses something called tuple types for their `chains` and i wanted to stick to their types as closely as possible for maintenance purposes. 

They force to use one or more chains: [readonly [Chain, ...Chain[]]](https://rc.wagmi.sh/core/api/createConfig#chains)

I think it's fine for now since this won't impact anything the rainbowkit side. I think having `mainnet` as a default chain is fine for now. Let me know if you have any thoughts 🙏